pkg/services/apiserver/auth/authorizer/provider.go

```diff
@@ -21,6 +22,7 @@ type GrafanaAuthorizer struct {
 func NewGrafanaAuthorizer(cfg *setting.Cfg, orgService orgsvc.Service) *GrafanaAuthorizer {
 	authorizers := []authorizer.Authorizer{
 		&impersonationAuthorizer{},
+		authorizerfactory.NewPrivilegedGroups("system:masters"),
```

Comment:
any concerns if we always allow requests from the loopback's system:masters group?

Reply:
what does it take to get in the "system:masters" group?  with impersonation disabled, that seems pretty safe

Reply:
i think it's only possible via the loopback config